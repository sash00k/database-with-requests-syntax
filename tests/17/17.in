get records where student_id in (get records where group in (get records where course is "Программирование").groups).students