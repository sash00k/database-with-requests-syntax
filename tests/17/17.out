Student Иванов С. В. from group 22129, ММФ
Student Зуброва Д. А. from group 8126, ММФ